AptosSDK/Src/Client/Services/FaucetService.cs

```diff
@@ -0,0 +1,22 @@
+using System.Threading.Tasks;
+using Mirage.Aptos.SDK.DTO;
+
+namespace Mirage.Aptos.SDK.Services
+{
+	public class FaucetService
```

Comment:
Is it supposed to inherit BaseService? If no consider improving the naming.

Reply:
It isn't supposed to inherit BaseService but is also a service. The difference between it and others that inherited BaseService is they work with different clients.